platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/internal/DefaultExceptionContainer.java

```diff
@@ -16,19 +16,20 @@
 
 package org.gradle.tooling.events.problems.internal;
 
+import org.gradle.tooling.Failure;
 import org.gradle.tooling.events.problems.ExceptionContainer;
 
 import javax.annotation.Nullable;
 
 public class DefaultExceptionContainer implements ExceptionContainer {
-    private final RuntimeException exception;
+    private final Failure exception;
 
-    public DefaultExceptionContainer(@Nullable RuntimeException exception) {
+    public DefaultExceptionContainer(@Nullable Failure exception) {
         this.exception = exception;
     }
 
     @Override
-    public RuntimeException getException() {
+    public Failure getException() {
```

Comment:
🤔 Can the names of these methods be changed to `getFailure` as well, as it is no longer returning an exception?